backend/src/auth/models.py

```diff
@@ -1,45 +1,43 @@
 import uuid
 from datetime import datetime
-from typing import TYPE_CHECKING, Optional
 
 from sqlalchemy import UUID, Boolean, Column, DateTime, String
-from sqlalchemy.orm import Mapped, relationship
+from sqlalchemy.orm import relationship
 
 from src.core.database import Base
 
-if TYPE_CHECKING:
-    from src.users.models import Profile
-
 
 class User(Base):
     __tablename__ = "users"
 
-    id: uuid.UUID = Column(
+    id = Column(
         UUID(as_uuid=True),
         primary_key=True,
         default=uuid.uuid4,
     )
-    username: str = Column(String(length=100), nullable=False)
-    email: str = Column(String(length=100), unique=True, nullable=False)
-    password: str = Column(String(length=255), nullable=False)
-    created_at: datetime = Column(DateTime, default=datetime.utcnow())
-    updated_at: Optional[datetime] = Column(
+    username = Column(String(length=100), nullable=False, unique=True)
+    email = Column(String(length=100), unique=True, nullable=False)
+    password = Column(String(length=255), nullable=False)
```

Comment:
You dont need to add nullable = False, it is Falsy by default

Reply:
> You dont need to add nullable = False, it is Falsy by default

oh ok. And this got a required ? what if i want to do blank sometimes?